1080i/Variables.xml

```diff
@@ -318,7 +318,8 @@
 		<value condition="ListItem.IsResumable">overlays/watched/resume.png</value>
 		<value condition="ListItem.IsCollection">overlays/set.png</value>
 		<value condition="!String.IsEmpty(ListItem.Overlay)">$INFO[ListItem.Overlay]</value>
-		<value condition="ListItem.IsFolder + Container.Content(files)">overlays/folder.png</value>
+		<value condition="ListItem.IsFolder + Container.Content(files) + !Listitem.IsParentFolder">overlays/folder.png</value>
+		<value condition="ListItem.IsFolder + Container.Content(files) + Listitem.IsParentFolder">overlays/folderback.png</value>
```

Comment:
revert this whole cange and just add
<value condition="ListItem.IsParentFolder">overlays...</value>